packages/block-editor/src/components/inserter/index.js

```diff
@@ -96,7 +120,7 @@ class Inserter extends Component {
 				onToggle={ this.onToggle }
 				expandOnMobile
 				headerTitle={ __( 'Add a block' ) }
-				renderToggle={ this.renderToggle }
+				renderToggle={ toggle }
```

Comment:
I'd keep `this.renderToggle`, as it's IMO easier to trace variables across the file.

Reply:
yes, good catch, looks like a leftover ...